setuptools/msvc.py

```diff
@@ -1039,7 +1039,13 @@ def VCTools(self):
 
         elif self.vs_ver >= 15.0:
             host_dir = (
-                r'bin\HostX86%s' if self.pi.current_is_x86() else r'bin\HostX64%s'
+                r'bin\HostX86%s'
+                if self.pi.current_is_x86()
+                else (
+                    r'bin\HostARM64%s'
+                    if self.pi.current_cpu == 'arm64'
+                    else r'bin\HostX64%s'
+                )
             )
```

Comment:
Not a maintainer here, but personally I find if-else chains to be much more readable and concise than nested ternaries:
```py
            if self.pi.current_is_x86():
                host_dir = r'bin\HostX86%s'
            elif self.pi.current_cpu == 'arm64':
                host_dir = r'bin\HostARM64%s'
            else:
                host_dir = r'bin\HostX64%s'
```

Reply:
Updated.